client/app/components/dialog-content/dialog-content.html

```diff
@@ -20,7 +20,7 @@ <h2 class="text-capitalize no-wrap">{{ ::vm.dialog.label }}</h2>
                 ng-repeat="dialogField in ::dialogGroup.dialog_fields "
                 ng-init="inputTitle = dialogField.description"
                 class="form-horizontal">
-                <div class="form-group">
+                <div ng-show="dialogField.visible" class="form-group">
```

Comment:
@AllenBW I considered the option, but thought it was a purpose, having an invisible field in the dialog, but still sending its value. I'm not 100% sure about it though... I'm OK with changing it, if it's correct.